### ISSUE TITLE
Civilians run from patrolling aliens

### ISSUE THREAD

**Issue by pledbrook** [X2CommunityCore/X2WOTCCommunityHighlander#666]
See [lwotc#705](https://github.com/long-war-2/lwotc/issues/705) for an example of this. It's not really a problem for the base game, but in LWOTC and RedDobe's Yellow Alert Gameplay mod, this will put the enemy units in sound range into yellow alert.

**Comment by RedDoby** [comment]
Yes,  this would affect yellow alert gameplay.  I never noticed this but good find!

**Comment by RedDoby** [comment]
I know in lw2 they disabled the default civilian yellow behavior,  which I have done in yellow alert gameplay.  That may be the fix you need for this, but I am not sure.  It’s under the civilian behavior class.

**Comment by pledbrook** [comment]
Interesting. That could be it. Does that not disable civilians yelling when they're running from XCOM?

**Comment by MrNiceUK** [comment]
I highly disapprove of the "bug-basegame" tag for the issue. "Civilians run from patrolling aliens" is just a statement of fact, not a bug. Now if you want to claim that some underlying mechanics are bugged, then make it clear what they are in the issue title & description.

**Comment by pledbrook** [comment]
All bug reports are statements of fact about how the game behaves. "Civilians running from patrolling aliens" *is* a mechanic. Now, you may dispute that that mechanic is a bug, which is fair enough. I'm not sure at this point how to call it. If it is indeed connected with the implementation that RedDoby mentions, then here's tracktwo's comment on it:

> // tracktwo - BTRunCompletePreExecute: Add configurable flag to enable/disable the civilian 'yell' action.
> //            For LW2 this is disabled - hostile civvies start in red alert in most circumstances or quickly enter
> //            red alert. This yell ability seems incomplete (doesn't appear to work at all and generates a lot of
> //            log spam.

In particular, "doesn't appear to work at all" implies that it's bugged. Still, will remove the label since that description is contested.

**Comment by RedDoby** [comment]
> Interesting. That could be it. Does that not disable civilians yelling when they're running from XCOM?

I am not entirely sure what it does,  but I took it out and added the new yell alerts, and everything seems to be working as it should.

**Comment by Iridar** [comment]
Assigning this issue to @RedDoby , as they seem to be the go-to person for things related to yellow alert. Please correct me if that was too presumptuous.

**Comment by pledbrook** [comment]
Funnily enough, a player encountered this just yesterday with LWOTC because I had missed adding my fix to the LWOTC highlander.

**Comment by pledbrook** [comment]
And apparently I never submitted it as a pull request. My bad!
